ui/src/stores/userStore.ts

```diff
@@ -110,7 +110,6 @@ export const useUserStore = defineStore("user", {
     },
     async updateCurrentUser(user: any) {
       this.currentUser = user;
-      await this.getAllFromUserDatabase();
```

Comment:
Need to make sure userStore.getAllFromUserDatabase is called after user is changed. So both in Login.vue and Logout.vue. Might be other places too